corehq/apps/export/templates/export/customize_export.html

```diff
@@ -548,22 +559,30 @@
                                         </td>
                                         <td><input class="input-xlarge" type="text" data-bind="value: display" /></td>
                                         {% if request|toggle_enabled:'SPLIT_MULTISELECT_EXPORT' %}
-                                        <td class="input" data-bind="if: allOptions">
-                                            <span data-bind="if: hasNewOptions" class="pull-right">
-                                                <span class="icon-asterisk" title="{% trans "New options available" %}"></span>
-                                            </span>
-                                            <select data-bind="
-                                                value: doc_type,
-                                                foreach: $root.column_type_options
-                                            ">
-                                                <option data-bind="value: value, text: label"></option>
-                                            </select>
-                                            <div data-bind="if: doc_type() === 'SplitColumn'">
-                                                <select class="select2" data-bind="options: allOptions, selectedOptions: options" multiple="true">
+                                        <td class="input">
+                                            <div data-bind="if: !allOptions()">
+                                                <select disabled>
+                                                    <option>{% trans "plain" %}</option>
+                                                </select>
+                                            </div>
+                                            <div data-bind="if: allOptions">
```

Comment:
`allOptions` is a knockout observable object. Evaluating it returns the observables value which is required if you want to do any calculations based on it e.g `not` but knockout automatically `unwraps` observables in its bindings to so there is no need to evaluate the observable if you don't need the raw value.